README.md

```diff
@@ -37,11 +37,11 @@ You can see more documentations in [here](http://go-chassis.readthedocs.io/en/la
 git clone git@github.com:go-chassis/go-chassis.git
 ```
 
-3. Use [glide](https://github.com/Masterminds/glide) to download dependencies
+3. Use [dep](https://github.com/golang/dep) to download dependencies
 
 ```sh
-cd go-chassis 
-glide install
+cd go-chassis
+dep ensure
```

Comment:
https://go-chassis.readthedocs.io/en/latest/getstarted/install.html

here also need to change